st2common/tests/unit/test_util_keyvalue.py

```diff
@@ -141,7 +142,7 @@ def test_get_key(self, deseralize_key_value, KeyValuePair):
         KeyValuePair.get_by_scope_and_name().value = value
         deseralize_key_value.return_value = value
 
-        result = kv_utl.get_key(key=key, user=USER, decrypt=decrypt)
+        result = kv_utl.get_key(key=key, user=auth_db.UserDB(name=USER), decrypt=decrypt)
```

Comment:
Do you need `.name` here on `auth_db.UserDB(name=USER)`?

Reply:
Nope. This unit test is doing a shortcut and passing in a UserDB object.